client/src/Constants.js

```diff
@@ -0,0 +1,10 @@
+/* Store constant values here */
+
+export const minPasswordLength = 6;
+export const minPasswordErrMsg = `Your password must be at least ${minPasswordLength} characters long`;
+export const takenEmailErrMsg = "That email is already taken";
```

Comment:
I'm not sure if I see a reason for storing these as constants, since they won't be used more than once in the app

Reply:
I wanted it to be somewhere I can easily find that way I won't have to search for it. But you're right it's not really necessary if its only being used in one file.